### PR TITLE
Cloudwatch dev env fixes

### PR DESCRIPTION
**Pull request by ben851**
# Summary | Résumé

Adding fixes for the dev env delete/recreate with cloudwatch enabled

## Related Issues | Cartes liées

* https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/480

## Before merging this PR

Read code suggestions left by the
[cds-ai-codereviewer](https://github.com/cds-snc/cds-ai-codereviewer/) bot. Address
valid suggestions and shortly write down reasons to not address others. To help
with the classification of the comments, please use these reactions on each of the
comments made by the AI review:

| Classification      | Reaction | Emoticon |
|---------------------|----------|----------|
| Useful              | +1       | 👍        |
| Noisy               | eyes     | 👀        |
| Hallucination       | confused | 😕        |
| Wrong but teachable | rocket   | 🚀        |
| Wrong and incorrect | -1       | 👎        |

The classifications will be extracted and summarized into an analysis of how helpful
or not the AI code review really is.

## Test instructions | Instructions pour tester la modification

Tested locally - validate next week that dev env creates properly.

## Release Instructions | Instructions pour le déploiement

None.

## Reviewer checklist | Liste de vérification du réviseur

* [x] This PR does not break existing functionality.
* [x] This PR does not violate GCNotify's privacy policies.
* [x] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
* [x] This PR does not significantly alter performance.
* [x] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: scripts/deleteEnvironment.sh [hunk 1]
```diff
@@ -96,13 +96,14 @@
...
-aws logs delete-query-definition --query-definition-id $(aws logs describe-query-definitions --query 'queryDefinitions[?name==`Lambda Statistics / system_status`]'.queryDefinitionId --output text)
-aws logs delete-query-definition --query-definition-id $(aws logs describe-query-definitions --query 'queryDefinitions[?name==`Lambda Statistics - ses_to_sqs_email_callbacks`]'.queryDefinitionId --output text)
-aws logs delete-query-definition --query-definition-id $(aws logs describe-query-definitions --query 'queryDefinitions[?name==`Lambda Statistics - ses_receiving_emails`]'.queryDefinitionId --output text --region us-east-1) --region us-east-1
-aws logs delete-query-definition --query-definition-id $(aws logs describe-query-definitions --query 'queryDefinitions[?name==`API / Services going over daily rate limits`]'.queryDefinitionId --output text)
+aws logs delete-log-group --log-group-name '/aws/eks/notification-canada-ca-dev-eks-cluster/cluster'
+aws logs delete-log-group --log-group-name '/aws/rds/cluster/notification-canada-ca-dev-cluster/postgresql'
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/deleteEnvironment.sh:100]: No thanks

File: scripts/deleteEnvironment.sh [hunk 2]
```diff
@@ -96,13 +96,14 @@
...
+
+QUERIES=$(aws logs describe-query-definitions --query 'queryDefinitions[].queryDefinitionId' --output text)
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/deleteEnvironment.sh:102]: No thanks

File: aws/eks/cloudwatch_log.tf [hunk 3]
```diff
@@ -2,11 +2,6 @@
 # AWS EKS Cloudwatch groups
 ###
 
-import {
-  to = aws_cloudwatch_log_group.blazer[0]
-  id = "blazer"
-}
-
```
Comment (sastels) [inline aws/eks/cloudwatch_log.tf:9]: We don't need this for staging / prod?

**Review by sastels — APPROVED** [review]

File: scripts/deleteEnvironment.sh [hunk 4]
```diff
@@ -138,5 +139,11 @@
   echo "Done."
 done
 
+QUERIES=$(aws logs describe-query-definitions --query 'queryDefinitions[].queryDefinitionId' --output text)
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/deleteEnvironment.sh:142]: Not necessary for what we're doing here.

**Pull request merged**